index.html

```diff
@@ -110,6 +110,83 @@ <h3 class="my-works-project-title">Tonic</h3>
+        const li3 = document.createElement('li');
+        li3.setAttribute('class','title-header-general');
+        li3.innerHTML += 'Back End Dev';//<li class="title-header-general">Back End Dev</li>
+        const li4 = document.createElement('li');
+        li4.innerHTML += `<img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator">`;//<li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+        const li5 = document.createElement('li');
+        li5.setAttribute('class','title-header-general');
+        li5.innerHTML += '2015';//<li class="title-header-general">2015</li>
+        ul.append(li1,li2,li3,li4,li5); 
+        headerTitlesWorks.appendChild(ul);
+        const para = document.createElement('p');
+        para.setAttribute('class','project-description');//<p class="project-description">
+        para.innerHTML += 'A daily selection of privately personalized reads; no accounts or sign-ups required.'; 
+        headerTitlesWorks.appendChild(para);
+        headerTitles.appendChild(headerTitlesWorks);
+        const section = document.createElement('section');
+        section.setAttribute('class','project-languages');//<section class="project-languages">
+        const sectionUl = document.createElement('ul');
+        sectionUl.setAttribute('class','my-work-languages');//<ul class="my-work-languages">
+        const sectionUlLi1 = document.createElement('li');
+        sectionUlLi1.setAttribute('class','coding-languages');
+        sectionUlLi1.innerHTML += 'html';//<li class="coding-languages">html</li>
+        const sectionUlLi2 = document.createElement('li');
+        sectionUlLi2.setAttribute('class','coding-languages');
+        sectionUlLi2.innerHTML += 'css';//<li class="coding-languages">css</li>
+        const sectionUlLi3 = document.createElement('li');
+        sectionUlLi3.setAttribute('class','coding-languages');
+        sectionUlLi3.innerHTML += 'css';//<li class="coding-languages">javascript</li>
+        sectionUl.append(sectionUlLi1, sectionUlLi2, sectionUlLi3);
+        section.appendChild(sectionUl);
+        headerTitles.appendChild(section);
+        const sectionProject = document.createElement('section');
+        sectionProject.setAttribute('class','project-more');//<section class="project-more">
+        const sectionProjectBtn = document.createElement('button');
+        sectionProjectBtn.setAttribute('class', 'works-see-more');
+        sectionProjectBtn.id = 'tonic-project-btn'; //<button class="works-see-more" type="button" id="tonic-project-btn">See Project</button>
+        sectionProjectBtn.innerHTML += 'See Project';
+        sectionProject.appendChild(sectionProjectBtn);
+        headerTitles.appendChild(sectionProject);
+        myworksDiv.appendChild(headerTitles);
+        document.getElementById('portfolio').appendChild(myworksDiv); 
+    </script>
+    -->
+
         <div class="my-works">
```

Comment:
- [ ] Great job adding your project modals dynamically, however as suggested by the previous reviewer, I also suggest that you also need to add your project cards themselves dynamically as you did for the project modal.
- [ ] I kindly suggest that you remove the project cards you added manually in your HTML and add them dynamically through JavaScript as you did for the modal

![image](https://user-images.githubusercontent.com/52934128/196008699-031b1d43-7289-46e9-8ac8-034303254968.png)